### PR TITLE
Math magicians: components

### PR DESCRIPTION
**Pull request by bill7pearl**
# In this PR I have:
- Deleted all the boilerplate from CRA (text, images, styles).
- Created a directory called components.
- Inside components, created a new Calculator.js file.
- In Calculator.js, created a React component that matches the design provided.

### CODE REVIEW

**Comment by TedLivist** [comment]
## STATUS: INVALID CODE REVIEW REQUEST 🛑 🛑

Hi @bill7pearl,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### 1. Kindly fix your linter error.
### 2. Kindly ensure you format your README using [the Microverse template](https://github.com/microverseinc/readme-template). All table of contents should be present and working correctly

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/components/Wrapper.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+/* eslint-disable react/prop-types */
```
Comment (Deepakdanger) [inline src/components/Wrapper.js]: - [ ] Kindly remove it & solve it manually

File: src/components/CalcContext.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+/* eslint-disable react/prop-types */
```
Comment (Deepakdanger) [inline src/components/CalcContext.js]: - [ ] Kindly remove it & solve it manually

File: src/components/ButtonBox.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
+/* eslint-disable react/prop-types */
```
Comment (Deepakdanger) [inline src/components/ButtonBox.js]: - [ ] Kindly remove it & solve it manually

File: src/components/Button.js [hunk 4]
```diff
@@ -0,0 +1,114 @@
+/* eslint-disable react/button-has-type */
+/* eslint-disable react/prop-types */
```
Comment (Deepakdanger) [inline src/components/Button.js]: - [ ] Kindly remove it & solve it manually

File: src/App.js [hunk 5]
```diff
@@ -1,28 +1,34 @@
-import logo from './logo.svg';
-import './App.css';
+/* eslint-disable react/no-array-index-key */
+/* eslint-disable import/no-extraneous-dependencies */
```
Comment (Deepakdanger) [inline src/App.js]: - [ ] Kindly remove it & solve it manually

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @bill7pearl  

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight :100: 
- No linter error :heavy_check_mark: 
- Website looks professional :+1: 

## Required Changes ♻️
- [ ] The button `0` is twice in length, kindly change it according to the original design.
- your design


Screenshot from 2023-01-19 20-06-25


- Original design
Screenshot from 2022-11-10 00-26-21





- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] Kindly use the same color as `=` button as in the original design, after approval you can advance your design calculator in various ways.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 5 above)
Comment (alexander16108) [inline src/App.js]: - [ ]  Kindly remove it & solve it manually

File: src/components/Button.js (hunk 4 above)
Comment (alexander16108) [inline src/components/Button.js]: - [ ] Kindly remove it & solve it manually

File: src/components/ButtonBox.js (hunk 3 above)
Comment (alexander16108) [inline src/components/ButtonBox.js]: - [ ] Kindly remove it & solve it manually

File: src/components/CalcContext.js (hunk 2 above)
Comment (alexander16108) [inline src/components/CalcContext.js]: - [ ] Kindly remove it & solve it manually

File: src/components/Wrapper.js (hunk 1 above)
Comment (alexander16108) [inline src/components/Wrapper.js]: - [ ] Kindly remove it & solve it manually

**Review by alexander16108 — CHANGES_REQUESTED** [review]
Hi @bill7pearl , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights

- No linter error ✔️
- Website looks professional 👍

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AnselemOdims — APPROVED** [review]
### Hi  @bill7pearl 🙋‍♀️, 
Good job in making the required changes 👏 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**